Dockerfile

```diff
@@ -0,0 +1,14 @@
+# run with:
+# docker build -t lazygit
+# docker run -it lazygit:latest
+
+FROM golang:alpine
+
+RUN apk add git
+RUN apk add xdg-utils
```

Comment:
tbh i would squash these two `RUN`s to one of this form:
`RUN apk add -U git xdg-utils`
so we can have one layer with all dependencies needed

the `-U` flag is for apk's cache updating

Reply:
updated :)